asllib/carpenter/ASTEnums.ml

```diff
@@ -0,0 +1,427 @@
+(*
+ * SPDX-FileCopyrightText: Copyright 2022-2023 Arm Limited and/or its affiliates <open-source-office@arm.com>
```

Comment:
Did you make any changes in this file in 2024? Then you need to extend the copyright. This applies for all copyright headers.

```suggestion
 * SPDX-FileCopyrightText: Copyright 2022-2024 Arm Limited and/or its affiliates <open-source-office@arm.com>
```